docs/src/development/local/ddev.md

```diff
@@ -3,9 +3,10 @@ title: Use DDEV for local development
 sidebarTitle: DDEV
 description: Use DDEV to set up local development environments.
 weight: 1
+sectionBefore: Integrated environments
 ---
 
-[DDEV](https://ddev.readthedocs.io/en/stable/) is an open-source tool for local PHP development environments.
+[DDEV](https://ddev.readthedocs.io/en/stable/) is an open-source tool for local PHP and Node.js development environments.
```

Comment:
Node.js apps is not supported yet.

Reply:
Is that just the add-on? The repository description included Node.js, which is why I added it: https://github.com/drud/ddev

Reply:
AFAIK it doesn't support Node.js webapps yet. Right @rfay ?

Reply:
Node.js is significantly supported in DDEV, including webapps and other uses, but there is no mapping from Platform's support to DDEV's support.

Reply:
I'm not sure how to present that exactly. Guess I can leave out Node.js, just seemed unfortunate not to mention the support.